### PR TITLE
Move decimal files to separate repository.

### PR DESCRIPTION
**Pull request by alittlezz**
- Move the decimal files to separate repository.
- Separate the objects in different files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/target
```
Comment (y2kappa) [inline .gitignore:1]: there must be a `ratio` type as well

Comment (alittlezz) [reply .gitignore:1]: you're right, gonna add it now

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[package]
```
Comment (y2kappa) [inline Cargo.toml:1]: also, please add the rust github actions -> go to "Actions" and search for Rust, use this file
https://github.com/hubble-markets/hubble/blob/master/.github/workflows/rust.yml

but remove the "continue-on-error" to be false

File: .gitignore (hunk 1 above)
Comment (y2kappa) [reply .gitignore:1]: also, please add a README.md and say that it was taken from here https://github.com/solana-labs/solana-program-library/blob/5e320ba9768cca8ca751a4db26786ed13bb0d40f/token-lending/program/src/math/decimal.rs with some of our own additions

You can give some examples of how it's being used in Readme if you wish

File: Cargo.toml (hunk 2 above)
Comment (alittlezz) [reply Cargo.toml:1]: done!

**Pull request merged**